zone.go

```diff
@@ -0,0 +1,244 @@
+package dns
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"sort"
+	"strings"
+)
+
+// AddDomain adds origin to s if s is not already a FQDN.
+func AddDomain(origin, s string) string {
```

Comment:
[ Quoting <notifications@github.com> in "Re: [dns] Added new functions: Trim..." ]
>> @@ -0,0 +1,244 @@
>> +package dns
>> +
>> +import (
>> +	"bytes"
>> +	"fmt"
>> +	"io"
>> +	"log"
>> +	"sort"
>> +	"strings"
>> +)
>> +
>> +// AddDomain adds origin to s if s is not already a FQDN.
>> +func AddDomain(origin, s string) string {
>
>Could it be similar to IDN when things live within this library but in separate import path?

Hmmm, hadn't considered that. Nice idea.

/Miek

--
Miek Gieben